Earthfile

```diff
@@ -79,11 +79,18 @@ smoke-test:
   FROM +smoke-init
   WORKDIR /docusaurus2
   RUN yarn global add fs-extra jest
-  COPY ./tests/e2e/specs ./__tests__/e2e/specs
+  COPY ./tests/e2e/specs/cli.spec.js ./__tests__/e2e/specs/cli.spec.js
   COPY ./tests/helpers ./__tests__/helpers
   COPY ./tests/e2e/jest.config.js ./jest.config.js
   ENV NODE_ENV=ci
   RUN node --expose-gc /usr/local/bin/jest --logHeapUsage --runInBand
+  COPY ./tests/e2e/docusaurus2-graphql-doc-generator-multi-instance.config.js ./docusaurus2-graphql-doc-generator-multi-instance.config.js
+  COPY ./tests/e2e/docusaurus2-graphql-doc-generator-multi-instance-1.config.js ./docusaurus2-graphql-doc-generator-multi-instance-1.config.js
+  COPY ./tests/e2e/docusaurus2-graphql-doc-generator-multi-instance-2.config.js ./docusaurus2-graphql-doc-generator-multi-instance-2.config.js
```

Comment:
Files `./tests/e2e/*.config.js` should be located under `tests/__data__`.

Reply:
So should we move the existing ones `docusaurus2-graphql-doc-generator.config.js` and `docusaurus2-graphql-doc-generator-groups.config.js` as well ? My understanding was that `__data__` folder contains the raw schemas and some markdown files, while configuration is considered to be code.

Reply:
Good catch! `__data__` should contain everything that is not a test.

Reply:
Alright, addressed in 0d26176.